### PR TITLE
Use Grid to improve GcnEventPage layout

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds Grid to GcnEventPage.

### CODE REVIEW

File: static/js/components/GcnEventPage.jsx [hunk 1]
```diff
@@ -263,203 +269,210 @@ const GcnEventPage = ({ route }) => {
...
+            >
+              <Typography className={styles.accordionHeading}>
+                Light curve
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails>
+              <div className={styles.gcnEventContainer}>
+                {gcnEvent.lightcurve && (
+                  <div>
+                    {" "}
```
Comment (guynir42) [inline static/js/components/GcnEventPage.jsx:350]: Do we still need this funky spacing here?

**Review by guynir42 — APPROVED** [review]
Looks good although there may be a few instances of `{" "}` that may or may not be needed with the new grid.

File: static/js/components/GcnEventPage.jsx [hunk 2]
```diff
@@ -263,203 +269,210 @@ const GcnEventPage = ({ route }) => {
...
-          >
-            <Typography className={styles.accordionHeading}>
-              Sources within localization
-            </Typography>
-          </AccordionSummary>
-          <AccordionDetails>
-            {gcnEventSources?.sources.length === 0 ? (
-              <Typography variant="h5">None             </Typography>
-            ) : (
+    <Grid container spacing={2} className={styles.source}>
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply static/js/components/GcnEventPage.jsx:272]: Its in there but I think that is vertical. This gives extra space on the horizontal seems like.

File: static/js/components/GcnEventPage.jsx (hunk 1 above)
Comment (mcoughlin) [reply static/js/components/GcnEventPage.jsx]: Yeah this was a holdover from maybe the first iteration of this page. Deleting.

**Comment by mcoughlin** [comment]
Thanks @guynir42 .

**Pull request merged**